src/app/core/menu.service.ts

```diff
@@ -690,4 +706,13 @@ export class MenuService {
     }
   }
 
+  private resolveVoucherPermissions(voucherPermissions: VoucherPermissions[]): ResolvedVouchersPermissions {
+    const buy = voucherPermissions.find(config => config.buy) ? true : false;
```

Comment:
```suggestion
    const buy = !!voucherPermissions.find(config => config.buy);
```
Apply the same for others.